### ISSUE TITLE
mcce version installed via conda

### ISSUE THREAD

**Issue by mbatgh** [GunnerLab/Stable-MCCE#300]
Hi,

I successfully installed mcce using conda as suggested here:
https://gunnerlab.github.io/Stable-MCCE/quick/ 
which gives me mcce version 1.1.0 (at least it tells me so when I run the command with the --version flag)

Now there is a paper from 2009 called "MCCE2: Improving protein pKa calculations with extensive 
side chain rotamer sampling".

also on https://sites.google.com/site/mccewiki it says:
"Current development focuses are on the following 4 aspects: Integrating the new version 
of DelPhi into MCCE 3.5..."

and on https://github.com/GunnerLab  there is a repository called:
Zoom-Link-for-meeting-MCCE4

so it looks as if there were a number of versions around (at least 1, 2, 3, and 4) and conda installation 
leaves me with the seemingly oldest version There is also a repository called Develop-MCCE, but here 
the last update appears to be four years old, and documentation is rather sparse.

Is there a way to obtain a newer/the most recent version?

thanks!
Michael

**Comment by newbooks** [comment]
The most recent public version is Stable-MCCE compiled from source. The
compiling procedure is here:
https://mccewiki.levich.net/books/install-mcce/page/download-and-install-mcce

A newer MCCE version is under development with additional interfaces to
APBS and NextGenPB (the main developer is visiting Marilyn's lab this
summer).

**Comment by mbatgh** [comment]
thanks for the link to the compilation instructions, that looks useful!
I assume what I get compiling the source from Stable-MCCE is version 2, correct?
Is there anywhere something like "release notes" with some details about which algorithms and features are available in this current version?

**Comment by newbooks** [comment]
Yes, same algorithm as MCCE2, with more tools for analysis and some bug
fixes. The version under development will have regular release time and
notes. We shifted focus to the new version which will add new parameters,
alternative PBE solvers, and new algorithms. At this moment I can't say the
exact beta release date, but it is reasonable to expect within the end of
this year.
